src/bin/cli.ts

```diff
@@ -15,6 +16,8 @@ import writeErrorsToLog from './lib/write-errors-to-log'
 import { RequestBatch } from '../lib/offline-api/index'
 import { getConfig } from './lib/config'
 import ValidationError from '../lib/interfaces/errors'
+import { PlainClientAPI } from 'contentful-management'
+import { trim } from 'lodash'
```

Comment:
afaik, this is the only way to ensure only relevant imports.
```suggestion
import trim from 'lodash/trim'
```